test/unit/classification_test.rb

```diff
@@ -540,421 +540,389 @@ def setup
     assert enc['base'][key.key].nil?
   end
 
-  test "#enc should return correct merged override to host when multiple overrides for inherited hostgroups exist" do
-    FactoryBot.create(:setting,
-      :name => 'matchers_inheritance',
-      :value => true)
-    key = FactoryBot.create(:puppetclass_lookup_key, :as_smart_class_param, :omit => true,
-                             :override => true, :key_type => 'array', :merge_overrides => true,
-                             :path => "organization\nhostgroup\nlocation",
-                             :puppetclass => puppetclasses(:two))
-
-    parent_hostgroup = FactoryBot.create(:hostgroup,
-      :puppetclasses => [puppetclasses(:two)],
-      :environment => environments(:production))
-    child_hostgroup = FactoryBot.build(:hostgroup, :parent => parent_hostgroup)
-
-    host = FactoryBot.create(:host, :environment => environments(:production), :organization => taxonomies(:organization1),
-      :puppetclasses => [puppetclasses(:one)], :hostgroup => child_hostgroup)
-
-    as_admin do
-      LookupValue.create! :lookup_key_id => key.id,
-                          :match => "hostgroup=#{parent_hostgroup}",
-                          :value => ['parent'],
-                          :omit => false
-      LookupValue.create! :lookup_key_id => key.id,
-                          :match => "organization=#{taxonomies(:organization1)}",
-                          :value => ['org'],
-                          :omit => false
+  context 'with matchers_inheritance enabled' do
```

Comment:
Are changes in this test file related?

Reply:
yup, they were, but those can be done prior this change. Let me drop these